SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -30,4 +51,10 @@ class AppCenterViewController : NSViewController {
     UserDefaults.standard.set(userId, forKey: "userId")
     appCenter.setUserId(userId)
   }
+  
+  @IBAction func overrideCountryCode(_ sender: NSButton) {
+    if CLLocationManager.locationServicesEnabled() {
+      self.locationManager.startUpdatingLocation()
+    }
```

Comment:
We need to inform as a popup dialog that location service isn't enabled. Hard to get noticed if logs don't contain carrier country due to that reason.